### PR TITLE
Project 2: Building with Responsive Design, requesting for review 

### PR DESCRIPTION
**Pull request by ixboy**
Hello, this is our second project on the program its a copy of the  https://thenextweb.com/  website. we used all the previous concepts about HTML like flexbox, float, and grid, we also made it to be responsive in 3 different screen size types. please review our work. any feedback will highly be appreciated.

Thank You.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+    -HTML
+    -CSS
+
+## LIVE DEMO
+
+[Live Demo Link](https://fdi96.github.io/Building-with-responsive-desing/)
+
+##  AUTHORS
+
+👤 **Author1**
```
Comment (ioilmio) [inline README.md]: include your name here instead of Author

**Review by ioilmio — CHANGES_REQUESTED** [review]
# Hello @FdI96  @ixboy 👋🏼 
## Nice try on this one! Your project begins to take shape so keep it up like that! 👍🏼 

### You have to fix a couple of issues, let me list them:
#### Navigation Bar
- It should display different content depending on the screen size, it should contain links and not buttons;
image

- Medium version
 TNW logo appears at the leftmost part ([link to page](https://thenextweb.com/l))
- Small version 
 Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon)
image


#### Top 3 articles section
- Large version 
 It uses an overlay gradient for the big image
image
- Medium version
 TNW logo disappears

#### Latest news section
- Large version
 It aligns 4 articles per row
- Medium version
 It aligns 3 articles per row

#### Latest Funding section
- Large version
 It aligns 4 articles per row

### Fix this margin issue please (large version)
image


#### Categories (Apps, Gear, Tech, etc) section
- Large version
It aligns 3 category boxes per row
- Medium version 
It aligns 2 category boxes per row

#### Latest deals section
- Large version
 It aligns 4 deals per row
image

- Medium version changes
 It aligns 3 deals per row

#### Footer
 It contains different social media icons  ([link to page](https://thenextweb.com/l))
image
image



### General
- Refer to the Odin project for more detailed requirements about each section of the project
- Change font-family to adapt to the original page;
- Make all the linters checks pass
- Check better your top3 article section, I think your horizontal scroll comes from there...
- Check your medium screen size; I can't see the difference between large(1024px) and medium (768px) screen;
- Add your node_modules folder to .gitignore so it won't be pushed on GitHub(for what are you using it??)

### You can do it Team 💯  so don't be discouraged, is always your first review, and there's always room for improvements;
#### For any doubts or further info you can contact me on slack @illuminatosalvatore

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUESTED 🚦 ⚠️
Hello @ixboy and @ioilmio 🤝 🤝 🤝
👍 This is good to make it this far, you have tried to implement some changes as required by the previous TSE.

- [x] However, before adding more changes, kindly work on **ALL** the changes requested by the previous  TSE. Please refer to the [Odin project ](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design)for more detailed requirements about each section of the project. Also, check out the original page (https://thenextweb.com/) for further details.

Regards
Peris Ndanu
Feel free to slack me if you have any questions.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are e some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] The live preview link on the README links to a previous version of the website please update it, if I base my revision on that live preview, there are no changes made from the first review.

### [OPTIONAL]

These next observations are optional, but I must insist you work on them, the project right now looks sloppy and has these few issues that stop it from being a great project. I believe you can submit an awesome project :muscle: :sunglasses:.

- [ ] The media query dimension should be desktop version abote until 769px, tablet view 768px to 426px, mobile view 425 and below. 

- [ ] Element doesn't look as expected this might be related to the previous observation
image

- [ ] Element doesn't look as expected
image

- [ ] Element has a margin that should not exist
image

- [ ] Text and images should have a space between them
image

- [ ] Element have greater white space to the right, it should be equal to the left side.
image

### [Comments] :green_circle: 

- Use the media query inspector to observe the changes in the original website. 

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @ixboy and  @FdI96, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. You already fulfilled most of the requirements, kindly fix the following issues. 

[REQUIRED]

- [ ] Fix stylelint errors
- [ ] The navigation bar shouldn't have the TNW logo on large screens.

_Click the image to view it in full._

| Your Copy | Source |
| :---: | :---: |
|Screen Shot 2020-12-04 at 6 33 21 PM| Screen Shot 2020-12-04 at 6 42 51 PM|

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: :stop_sign: 
Hello team! Great job on this project. :muscle: However before moving on, I'd like to mention some issues.
### Mandatory

- [ ] Please fix the linter issue. There shouldn't be a linter issue on your pull request when you submit for a review.

- [x] Kindly remove the arrow on the right and put a search icon instead.
arrow

- [x] Kindly remove the unnecessary padding around the footer.
footer2


### Optional

- [ ] You can write your names under the authors' section. 

- [ ] You can provide the [MIT license](https://github.com/git/git-scm.com/blob/master/MIT-LICENSE.txt) link on your project. That will make your repo more professional.

It is a great job till now and keep up the good job. :+1: 
Please fix these issues and submit for a re-review.
If you have any questions about the review, feel free to reach me via Slack. :100: 
_Eyüp Sercan UYGUR_

**Comment by FdI96** [comment]
We already fix the requirements, but we are having a lot of issues to fix the StyleLint because when a change is done the same change appears as other errors that we have to fix, and so on.

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hey, team @ixboy and @FdI96  you guys did pretty well in this project 👏  👏  It almost looks like the original page. 

✔️ Good job on working on the TSE requirements

However, before we can mark it as complete lets fix three things. Don't feel discouraged when we keep requesting changes, we want you to be the best developers, now give me a smile 😄 

- [ ] The TNW logo is losing its pixels on the large version,  kindly check on that.  [link]( https://thenextweb.com/ )
View screenshot below 
c
 

- [ ] Stylelint errors are still failing, ensure you run `npx stylelint` on localhost to check if there are any errors. If you get errors on localhost, your linters will fail too. Check this [link ](https://github.com/microverseinc/linters-config/tree/master/html-css)on how to setup linters

- [ ] Make the following titles a heading
c

- [ ] Add an MIT licence to make it more professional

Thats all 😌 , kindly submit another review when done.

Regards
Peris Ndanu
Feel free to slack me if you have any questions

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @ & @**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] please fix your linters errors and request another code review.

### Optional 🟢 

 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by mimipeshy** [comment]
## STATUS: APPROVED 💯 🥳 

Hey, team 👋 👋  congratulations on working so hard on this project, it's therefore approved. Go ahead and merge. All the best in your next projects.

Regards
Peris Ndanu

**Pull request merged**